src/averageAges.js

```diff
@@ -1,5 +1,15 @@
 'use strict';
 
+const MALE = 'm';
+const FEMALE = 'f';
+const CENTURY = 100;
+
+function getAverengeAge(listOfPeople) {
+  const sumAge = listOfPeople.map((person) => person.died - person.born)
+    .reduce((acc, value) => acc + value, 0);
```

Comment:
Move the calculation from the `map` inside the `reduce`, so as not to go through the array twice.
Try using destructuring on the object.